packages/core/src/session_pool/session_utils.ts

```diff
@@ -1,13 +1,12 @@
-import type { Dictionary } from '@crawlee/types';
+import type { BrowserLike, Dictionary } from '@crawlee/types';
 import type { IncomingMessage } from 'node:http';
-import type { HTTPResponse } from 'puppeteer';
 import { Cookie } from 'tough-cookie';
 import { CookieParseError } from './errors';
 
 /**
  * @internal
  */
-export function getCookiesFromResponse(response: IncomingMessage | HTTPResponse | { headers: Dictionary<string | string[]> }): Cookie[] {
+export function getCookiesFromResponse(response: IncomingMessage | BrowserLike | { headers: Dictionary<string | string[]> }): Cookie[] {
```

Comment:
maybe `BrowserLikeResponse` would be better name?

Reply:
Sure